devpack/src/main/java/io/neow3j/devpack/Storage.java

```diff
@@ -160,224 +152,4 @@
     public static native Iterator<Entry<String, byte[]>> find(StorageContext context,
             String prefix);
 
-    /**
```

Comment:
can you briefly explain to me what exactly happened with all the Storage context and all the other calls? Will they be moved? or removed, completely?

Reply:
Good point. So, the implicit storage context methods were removed without replacement in the dotnet-devpack, which  I use as a reference implementation. That said, I agree that those methods did not have much use. No developer would use such methods because they always incur the overhead of fetching the `StorageContext` before doing the actual storage operation. That is more expensive than using the explicit storage context methods.